### ISSUE TITLE
appveyor git errors

### ISSUE THREAD

**Issue by isiyu** [mapbox/mapbox-unity-sdk#105]
@BergWerkGIS didn't get to anything conclusive today - looked like it was failing at the commit step. David and I have a couple of commits in appveyor that fail at the same spot. 

Can you take a look? thanks!

**Comment by BergWerkGIS** [comment]
Problem is that the `git` configuration or git version on AppVeyor seems to have changed.

Obviously AppVeyor has dropped its global configuration for (git) `username` and (git) `useremail`.
It's not possible to commit when those or dedicated ones for the repo are not set.

Before it was `user.name=appveyor`/`user.email=appveyor`:
```bash
$ git log -n 1
commit b68742b6b23b5bcc9ca3be087d5c2ec951b08a26
Author: appveyor <appveyor>
Date:   Tue May 16 18:46:50 2017 +0000

```

![image](https://cloud.githubusercontent.com/assets/4549888/26671759/faf796c2-46b6-11e7-8a22-b244833dfa0f.png)


Essentially we are creating a new repo at that step and before it took the global settings (which don't exist anymore).

Just like the error message says we would have to run
either (global):
```
git config --global user.email "you@example.com"
git config --global user.name "Your Name"
```
or (just for the new repo)
```
git config user.name "Your Name"
git config user.name "Your Name"
```

Which version we choose doesn't matter as that instance is terminated after the build.

Question: which username and email should we use?
Those show up in the commit history.
Do we have unity dedicated Github bot already?
Or should we use another existing Mapbox bot?

**Comment by BergWerkGIS** [comment]
Set git config to `appveyor/appveyor` as was before: https://github.com/mapbox/mapbox-unity-sdk/commit/29dd86bfcd801cb03594fb516f307b47736ff513

`git push` succeeded:
https://ci.appveyor.com/project/Mapbox/mapbox-unity-sdk/build/1.0.647#L1709

Branch [`appveyor-test`](https://github.com/mapbox/mapbox-unity-sdk/compare/appveyor-test?diff=split&name=appveyor-test) should be merged as I [added `mb-pages` and `gh-pages` to the backlist](https://github.com/mapbox/mapbox-unity-sdk/commit/2e4b037a334a3657b6f9de6cd0a46bd7fc847d1e) that we don't start builds when pushing to the doc branch:
![image](https://cloud.githubusercontent.com/assets/4549888/26674833/06c345b8-46c2-11e7-9e5e-cd30d36cad3b.png)

**Comment by david-rhodes** [comment]
Closed: #108
